templates/index.html

```diff
@@ -143,9 +143,9 @@
         </paper-toolbar>
         <div id="container" class="container" layout="" vertical="" center="">
           <x-schedule id="mainschedule" show="all"></x-schedule>
-          <div id="GithubUpdateBox" style="text-align:center; background-color:white;">
-            <p style="padding:12px; margin-bottom:12px; margin-top:0px; font-size:1.4rem; font-weight:300;">
-              <a style="color:black; text-decoration:none;" target="_blank">
+          <div id="GithubUpdateBox" class = "GithubUpdateBox">
+            <p class="GithubInnerP">
+              <a class="GithubTitleA" target="_blank">
```

Comment:
It's an <a> tag, which is used for hyperlinks - you should add a target here (probably to the main project page)